doc/cephfs/disaster-recovery-experts.rst

```diff
@@ -21,43 +21,46 @@ Advanced: Metadata repair tools
 Journal export
 --------------
 
-Before attempting dangerous operations, make a copy of the journal like so:
+Before attempting any dangerous operation, make a copy of the journal by
+running the following command:
 
-::
+.. prompt:: bash #
 
-    cephfs-journal-tool journal export backup.bin
+   cephfs-journal-tool journal export backup.bin
 
-Note that this command may not always work if the journal is badly corrupted,
-in which case a RADOS-level copy should be made (http://tracker.ceph.com/issues/9902).
+If the journal is badly corrupted, this command might not work. If the journal
+is badly corrupted, make a RADOS-level copy
+(http://tracker.ceph.com/issues/9902).
```

Comment:
@vshankar, Do you know how to make a RADOS-level copy of the journal? The linked tracker issue here seems not to explain how to do so.

Reply:
No, I have no idea and that tracker is ~10 years old. Wild guess is rados clie:

>    cppool <pool-name> <dest-pool>   copy content of a pool

But that's for the entire pool.

However, thinking more into this -- since this mechanism is likely not implemented, it should probably be removed from the docs.

Reply:
@vshankar Yeah. This whole section needs to be rethought. We'll do that over the next week. This section warns readers not to do anything unless they know what they're doing without really explaining how to learn how to be someone who knows what you're doing.

Reply:
Note to future @zdover23: I'm going to merge this and backport it with the grammar improvements in the text. A near-future PR will remove mention of this feature from the documentation. I would prefer for the removal of this feature to be separated out into its own PR.

Reply:
@vshankar, I've removed this reference here: https://github.com/ceph/ceph/pull/61426